includes/polling/os/ceragon.inc.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$hardware = trim(snmp_get($device, '1.3.6.1.4.1.4458.1000.1.1.30.0', '-OQv', '', ''), '"');
```

Comment:
please convert these to a snmp_multi_get_oid() call so we can save on snmp lookups.